nifi-registry-docker/dockerhub/README.md

```diff
@@ -0,0 +1,127 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Docker Image Quickstart
+
+## Capabilities
+This image currently supports running in standalone mode either unsecured or with user authentication provided through:
+   * [Two-Way SSL with Client Certificates](https://nifi.apache.org/docs/nifi-registry-docs/html/administration-guide.html#security-configuration)
+   * [Lightweight Directory Access Protocol (LDAP)](https://nifi.apache.org/docs/nifi-registry-docs/html/administration-guide.html#ldap_identity_provider)
+   
+## Building
+The Docker image can be built using the following command:
+
+    . ~/Projects/nifi-dev/nifi-registry/nifi-registry-docker/dockerhub/DockerBuild.sh
+
+This will attempt to build and tag an image matching the string in DockerImage.txt
+
+    dockerhub dchaffey$ cat DockerImage.txt
+    > apache/nifi-registry:0.1.0
+    docker images
+    > REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE
+    > apache/nifi-registry     0.1.0               751428cbf631        15 minutes ago      342MB
+    
+**Note**: The default version of NiFi-Registry specified by the Dockerfile is typically that of one that is unreleased if working from source.
+To build an image for a prior released version, one can override the `NIFI_REGISTRY_VERSION` build-arg with the following command:
+    
+    docker build --build-arg=NIFI_REGISRTY_VERSION={Desired NiFi-Registry Version} -t apache/nifi-registry:latest .
```

Comment:
typo in 'REGISTRY': `--build-arg=NIFI_REGISTRY_VERSION={Desired NiFi-Registry Version}`

Reply:
Sorry I missed the notification of your review @kevdoran 
All your suggestions are perfectly reasonable. I think that the pattern of passing environment variables is what is useful here, and expanding it to cover other requirements is a good idea.